tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -107,10 +106,46 @@ public function numericValueProvider()
      *
      * @expectedException \FINDOLOGIC\Export\Helpers\AttributeValueLengthException
      */
-    public function testCharacterLimitCausesException()
+    public function testAttributeValueCharacterLimitCausesException()
     {
         $value = implode('', array_fill(0, 16384, '©'));
 
         DataHelper::checkAttributeValueNotExceedingCharacterLimit('some attribute', $value);
     }
+
+    /**
+     * Test if item id character limit of data helper causes exception when called outside item class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\ItemIdLengthException
+     */
+    public function testItemIdCharacterLimitCausesException()
+    {
+        $id = implode('', array_fill(0, 256, 'a'));
+
+        DataHelper::checkItemIdNotExceedingCharacterLimit($id);
+    }
+
+    /**
+     * Test if group name character limit of data helper causes exception when called outside item class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\GroupNameLengthException
+     */
+    public function testGroupNameCharacterLimitCausesException()
+    {
+        $group = implode('', array_fill(0, 256, 'a'));
+
+        DataHelper::checkCsvGroupNameNotExceedingCharacterLimit($group);
+    }
+
+    /**
+     * Test if attribute key character limit of data helper causes exception when called outside item class.
+     *
+     * @expectedException \FINDOLOGIC\Export\Helpers\AttributeKeyLengthException
+     */
+    public function testAttributeKeyCharacterLimitCausesException()
+    {
+        $attributeKey = implode('', array_fill(0, 248, 'a'));
```

Comment:
Use a multibyte character for the test data so we cover that we use the right function for checking the string size.

Reply:
Created a Utility class and moved the generation of the multibyte character string into a static class